services/influxdb/service.go

```diff
@@ -1225,15 +1232,15 @@ func (c *influxdbCluster) startUDPListener(se subEntry, port string) (*net.UDPAd
 	conf.Buffer = c.udpBuffer
 	conf.ReadBuffer = c.udpReadBuffer
 
-	l := c.LogService.NewLogger(fmt.Sprintf("[udp:%s.%s] ", se.db, se.rp), log.LstdFlags)
-	service := udp.NewService(conf, l)
+	d := c.diag.WithUDPContext(fmt.Sprintf("udp:%s.%s", se.db, se.rp))
+	service := udp.NewService(conf, d)
 	service.PointsWriter = c.PointsWriter
 	err := service.Open()
 	if err != nil {
 		return nil, err
 	}
 	c.services[se] = service
-	c.logger.Println("I! started UDP listener for", se.db, se.rp)
+	c.diag.StartedUDPListener(fmt.Sprintf("%s.%s", se.db, se.rp))
```

Comment:
Again here, why not pass the db and rp strings to the method directly?

Reply:
fixed.